shell/platform/fuchsia/flutter/vulkan_surface.cc

```diff
@@ -50,33 +60,87 @@ bool VulkanSurface::CreateVulkanImage(vulkan::VulkanProvider& vulkan_provider,
       .collection = collection_,
       .index = 0,
   };
+  
+  const VkSysmemColorSpaceFUCHSIA kDefaultColorSpace = {
+    .sType = VK_STRUCTURE_TYPE_SYSMEM_COLOR_SPACE_FUCHSIA,
+    .pNext = nullptr,
+    .colorSpace = static_cast<uint32_t>(fuchsia::sysmem::ColorSpaceType::SRGB),
+  };
+
+  const VkImageFormatConstraintsInfoFUCHSIA kDefaultFormatInfo = {
+    .sType = VK_STRUCTURE_TYPE_IMAGE_FORMAT_CONSTRAINTS_INFO_FUCHSIA,
+    .pNext = nullptr,
+    .requiredFormatFeatures = 0u,
+    .flags = 0u,
+    .sysmemFormat = static_cast<uint64_t>(fuchsia::sysmem::ColorSpaceType::INVALID),
+    .colorSpaceCount = 1,
+    .pColorSpaces = &kDefaultColorSpace,
+  };
 
-  out_vulkan_image->vk_image_create_info = {
-      .sType = VK_STRUCTURE_TYPE_IMAGE_CREATE_INFO,
-      .pNext = &out_vulkan_image->vk_collection_image_create_info,
-      .flags = kVulkanImageCreateFlags,
-      .imageType = VK_IMAGE_TYPE_2D,
-      .format = kVulkanFormat,
-      .extent = VkExtent3D{static_cast<uint32_t>(size.width()),
-                           static_cast<uint32_t>(size.height()), 1},
-      .mipLevels = 1,
-      .arrayLayers = 1,
-      .samples = VK_SAMPLE_COUNT_1_BIT,
-      .tiling = VK_IMAGE_TILING_OPTIMAL,
-      .usage = kVkImageUsage,
-      .sharingMode = VK_SHARING_MODE_EXCLUSIVE,
-      .queueFamilyIndexCount = 0,
-      .pQueueFamilyIndices = nullptr,
-      .initialLayout = VK_IMAGE_LAYOUT_UNDEFINED,
+  const VkImageCreateInfo kDefaultImageCreateInfo = {
```

Comment:
this is not really a global constant as you're setting pnext and extent to local values. please rename to image_create_info or maybe better, fold it into the loop I'm suggesting below